### DISCUSSION TITLE
Override the default i18n messages

### DISCUSSION THREAD

**Discussion by Barabek88** [keycloakify/keycloakify#403] [Q&A]
Hi all. I have a problem. 
I need to override the  message for requiredAction.UPDATE_PASSWORD, but i can't do it.
Can you please tell me how to correctly write this **requiredAction.UPDATE_PASSWORD:  "my text"** to the I18n.ts file.
Thank you

**Comment by Barabek88** [comment]
Oh, I found a way. Everything is simple. I wrapped the string in quotes. **"requiredAction.UPDATE_PASSWORD": "my text"**
thanks for the work you've done

**Comment by garronej** [comment]
Thank you 😊
